From 8d00dd2c87099372b9a91f4d7a13bef6d8ac1144 Mon Sep 17 00:00:00 2001
From: Ahmad Abbas <ahmada@marvell.com>
Date: Thu, 14 Apr 2016 17:04:19 +0300
Subject: [PATCH 130/239] mv_ddr: make: updated the Makefile to work with yocto
 compiler

Added an option to get platform specific compilation flags from the calling
Makefile.
In addition added "nostdinc" link param in order to prevent linkage with
standard libraries.

Change-Id: I4df7c085d455920c374ac39c99a1daa541fa5dee
Signed-off-by: Ahmad Abbas <ahmada@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/29060
Reviewed-by: Shadi Ammouri <shadi@marvell.com>
Tested-by: Haim Boot <hayim@marvell.com>
---
 drivers/marvell/mv_ddr/Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/marvell/mv_ddr/Makefile b/drivers/marvell/mv_ddr/Makefile
index bdc5f6d..05f61eb 100644
--- a/drivers/marvell/mv_ddr/Makefile
+++ b/drivers/marvell/mv_ddr/Makefile
@@ -216,6 +216,7 @@ ifeq ($(MV_DDR4),y)
 INCPATH += ../mv_ddr4
 endif
 INCLUDE = $(addprefix -I, $(INCPATH))
+INCLUDE += $(PLAT_INCLUDES)
 
 LIBNAME = mv_ddr_lib.a
 LIB = $(OBJ_DIR)/$(LIBNAME)
@@ -224,7 +225,7 @@ LIBNAME4 = mv_ddr4_lib.a
 LIB4 = $(SRCDIR4)/$(LIBNAME4)
 endif
 
-CFLAGS = -Wall -Werror -Os -ffreestanding -mlittle-endian -g -gdwarf-2
+CFLAGS = -Wall -Werror -Os -ffreestanding -mlittle-endian -g -gdwarf-2 -nostdinc
 CFLAGS += -march=armv8-a -fpie $(INCLUDE) -D$(PLATFORM)
 
 CFLAGS += -DMV_DDR_ATF -DCONFIG_APN806 -DCONFIG_MC_STATIC
-- 
1.9.1

